lib/rules/no-invalid-role.js

```diff
@@ -100,6 +104,73 @@ const DISALLOWED_ELEMENTS = [
   'wbr',
 ];
 
+// List of roles taken from https://www.w3.org/WAI/PF/aria/roles#widget_roles
+// The following is a collated list of Widget, Document Structure and Landmark Roles
+const VALID_ROLES = [
```

Comment:
Just wondering, I noticed the `table` role, as one example, is not in this list. Any idea why? https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Row_Role

Reply:
Yes, I tried looking for table role in this [list](https://www.w3.org/WAI/PF/aria/roles) I am not able to find it , don't know why
May be @MelSumner can throw some light here.

Reply:
@bmish `table` is not a valid role. See the spec: https://www.w3.org/WAI/PF/aria/roles